apps/support/cmdline.h

```diff
@@ -106,11 +118,24 @@ Target lexical_cast(const Source &arg)
 
 static inline std::string demangle(const std::string &name)
 {
+#if !defined(_WIN32)
```

Comment:
Thanks for the Win32 fix. Looking over the broader use of this, I'm a little skeptical as to whether this demangling stuff actually pays for itself in the first place (I had forgotten this library does this stuff). But as long as it works, hey, why not :-)